R/browse.R

```diff
@@ -25,6 +26,8 @@
 #' browse_github_issues("backports", 1)
 #' browse_github_pulls("rprojroot")
 #' browse_github_pulls("rprojroot", 3)
+#' # Leads to 404, because usethis does not currently use GitHub Actions.
+#' browse_github_actions("usethis")
```

Comment:
I suspect it's better to `\dontrun{}` this then. In which case, I'd drag it down to the bottom, by itself.